examples/exit-handler-teams.yaml

```diff
@@ -0,0 +1,31 @@
+# This example shows how to hit a teams webhook on an exit handler.
+# In order to configure, create teams webhook url for it.
+# The documentation to do so is here: https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook
+# Replace TEAMS_WEBHOOK_URL_HERE on line 31 with that webhook url
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: exit-handler-slack-
+spec:
+  entrypoint: say-hello
+  onExit: exit-handler
```

Comment:
can you  update with lifecyclehook?

Reply:
@sarabala1979 looks like this was addressed. can we merge to provide this example to the community?

Reply:
bump: let's get this doc merged 👍